pkg/controller/stacks/install/installjob.go

```diff
@@ -72,6 +73,12 @@ func createInstallJob(i v1alpha1.StackInstaller, executorInfo *stacks.ExecutorIn
 		namespace = o.Namespace
 	}
 
+	imagePullSecrets := i.GetSpec().ImagePullSecrets
+	imagePullPolicy := i.GetSpec().ImagePullPolicy
+	if imagePullPolicy == "" {
```

Comment:
Do we want this default? This seems like it will change the default behavior of image pulls to be different from what it was before, especially for [implicit update cases](https://kubernetes.io/docs/concepts/containers/images/#updating-images).

Reply:
I'll give `corev1.PullPolicy("")` a try. This should result in the value being omitted.